modules/exploits/windows/smb/ms17_010_psexec.rb

```diff
@@ -103,7 +103,7 @@ def initialize(info = {})
   def exploit
     begin
       if datastore['SMBUser'].present?
-        print_status("Authenticating to #{datastore['RHOSTS']} as user '#{splitname(datastore['SMBUser'])}'...")
+        print_status("Authenticating to #{datastore['RHOST']} as user '#{splitname(datastore['SMBUser'])}'...")
```

Comment:
Is the `rhost` method available in this context?

Reply:
`datastore['RHOSTS']` (current) is not available. But `datastore['RHOST']` (suggested) is available, as demonstrated two lines below with:
`eternal_pwn(datastore['RHOST'])`

Or do you mean something different with "`rhost` **method**"?

Reply:
```
msf5 exploit(windows/smb/ms17_010_psexec) > pry
[*] Starting Pry shell...
[*] You are in exploit/windows/smb/ms17_010_psexec

[1] pry(#<Msf::Modules::Exploit__Windows__Smb__Ms17_010_psexec::MetasploitModule>)> method(:rhost).source_location
=> ["/rapid7/metasploit-framework/lib/msf/core/exploit/tcp.rb", 266]
[2] pry(#<Msf::Modules::Exploit__Windows__Smb__Ms17_010_psexec::MetasploitModule>)>
```

Reply:
I don't think it matters much. You'll find more consistency with the datastore, anyway.